src/github/push.ts

```diff
@@ -28,6 +34,18 @@ export const pushWebhookHandler = async (context: WebhookContext, jiraClient, _u
 		gitHubInstallationId
 	});
 
+	if (await booleanFlag(BooleanFlags.CONFIG_AS_CODE, false, jiraClient.baseURL)) {
```

Comment:
On a `push` webhook, we're now sending all modified files to the `updateRepoConfig()` function, which determines whether the config file has been changed and then reads the file and stores the config in the database.

Reply:
Won't it make more readable if we add check here to determines whether config file has been changed or not before calling `updateRepoConfig`?

Reply:
No, because this would mean that the push webhook processor would have to need to know what file to look for. That is information that only the user-config-service should have.

Reply:
The`user-config-service` can provide that information instead of having it in push webhook processor.

Reply:
Yeah, but that would mean we would have more code in the push webhook and less code in the service. I would rather have as little code as possible in the webhook processor because it should stay rather dumb.

That said, it's really a small issue and I wouldn't mind too much doing it the way you proposed if the code wasn't there already :)